data/tekton-pipelines/okd/windows10-installer.yaml

```diff
@@ -415,6 +401,8 @@ spec:
                   namespace: $(tasks.create-vm-from-template.results.namespace)
         - name: waitForSuccess
           value: "true"
+        - name: allowReplace
```

Comment:
Good idea, but does not fit the PR description.

Reply:
This is needed to run pipelines repeatedly, maybe I should create another PR for that?

Reply:
I think it is okay to keep it in this PR, but change the commit message accordingly please.